src/knownIssues/Test-SdnKIHostRootStoreNonRootCert.ps1

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Test-SdnKIHostRootStoreNonRootCert {
+    <#
+    .SYNOPSIS
+        Validate the Cert in Host's Root CA Store to detect if any Non Root Cert exist
+    .DESCRIPTION
+        Validate the Cert in Host's Root CA Store to detect if any Non Root Cert exist. Non Root Cert is the cert that Issuer not equal to Subject
+    .PARAMETER NcUri
+        Specifies the Uniform Resource Identifier (URI) of the network controller that all Representational State Transfer (REST) clients use to connect to that controller.
+    .PARAMETER ComputerName
+        Type the NetBIOS name, an IP address, or a fully qualified domain name of one or more remote computers.
+    .PARAMETER Credential
+        Specifies a user account that has permission to perform this action. The default is the current user.
+    .PARAMETER NcRestCredential
+        Specifies a user account that has permission to access the northbound NC API interface. The default is the current user.
+    .EXAMPLE
+        PS> <sample 1>
```

Comment:
Need to update the .EXAMPLE

Reply:
Added .EXAMPLE and removed un-used parameters comment